### ISSUE TITLE
Pause track at startup

### ISSUE THREAD

**Issue by timoloewe** [owais/remember-the-rhythm#1]
I'm loving your plug-in and it works fine for me. Thank you!

But what annoys me is that the last track automatically begins to play at Rhythmbox startup.
It would be great, if the track could be paused instead of directly playing when starting Rhythmbox.
Or perhaps a preference option to set the default behavior.

**Comment by owais** [comment]
That's on my radar. I think the best option is to handle everything without configuration. If the song was playing, it should play again upon startup and if the song was paused when rhythmbox closed, then track will be paused.

I think most of the people want the music player to start playing as soon as it starts. There is not much point in having such a plugin if users have to manually play the song after every restart.

What do you think?

**Comment by timoloewe** [comment]
I'm not sure about this. I used Songbird for a long time and it had a plug-in which behaved exactly like this (paused track will pause, playing track will play) and I didn't like it.

I'd like to be brought to the last played track, because I often want to continue listening to the same playlist, album or whatever. So I do not have to search through the library to "restore" my last session. To push the Play Button is not the problem. I rather don't want the program to automatically start playing because perhaps I want to listen to another song or playlist. It should start playing, when I've found what I was looking for.

And remembering the playback status (paused/playing) doesn't make sense in my opinion. Which person pauses a track because he wants it to be paused when he starts Rhythmbox the next time? When I exit Rhythmbox I'm not caring about its behavior on next startup, so why should it remember this?

So for me it makes most sense to alway pause upon startup. 

But obviously it depends on your listening behavior and your personal needs ;)

**Comment by owais** [comment]
Always pausing on startup can be a major regression for a lot of people me included. I'll make what I mentioned above default and add a configuration option to override that :) Should make everyone happy.

**Comment by timoloewe** [comment]
Yeah you're right, I'm probably not speaking for the majority of users...
But thank you very much, would love to see a configuration option for that!
Keep up, great work so far ;)
